roles/kube-burner/files/app-deployment.yml

```diff
@@ -21,6 +21,14 @@ spec:
           failureThreshold: 1
           timeoutSeconds: 60
           initialDelaySeconds: 30
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 8080
+          periodSeconds: {{ .livenessPeriod }}
+          failureThreshold: 1
+          timeoutSeconds: 15
+          initialDelaySeconds: 30
```

Comment:
@smalleni can you update the imagePullPolicy to `Always`, seems it's still using the old image?

Reply:
> @smalleni can you update the imagePullPolicy to `Always`, seems it's still using the old image?

@rsevilla87 do you have triggers in your perfApp repo that would have built a new container image from the PR?

Reply:
@rsevilla87 do you have triggers in your perfApp repo that would have built a new container image from the PR?